articles/azure/azs-how-create-service-fabric-portal.md

```diff
@@ -0,0 +1,218 @@
+
+1. Log in to the UKCloud Azure Stack portal.
+
+    For more detailed instructions, see the [*Getting Started Guide for UKCloud for Microsoft Azure*](azs-gs.md).
+
+2. In the favourites panel, select **Create a resource**.
+
+    ![New option in favourites panel](images/azsp_newmenu.png)
+
+3. In the *New* blade, select **Security + Identity**.
+
+   ![Security option in New blade](images/azs-browser-new-security.png)
+
+4. In the *New* blade, select **Key Vault**.
+
+   ![New Key Vault](images/azs-browser-new-key-vault.png)
+
+5. In the **Create key vault** blade, enter the following information:
+
+   - **Name** - The name of the key vault.
+
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   - **Resource Group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   - **Location** - This will be `frn00006`, which is the location of the Azure Stack.
+
+   - **Access policies** - Configure who can access the key vault and what permissions they have.
+
+     ![Create new key vault](images/azs-browser-create-key-vault.png)
+
+6. Click **Create**.
+
+7. You can monitor the progress of your Key Vault's deployment by clicking the **Notifications** icon.
+
+   ![!Notification showing vault deployment in progress](images/azsp_createvm_progress.png)
+
+### Adding a certificate to the Key Vault
+
+1. Once the key vault has been deployed, navigate to it by clicking **All services** in the favourites panel, then selecting **Key Vaults** under the *Security + Identity* section.
+
+2. Select your key vault from the list.
+
+3. In the **Settings** section of the key vault blade, select **Secrets**.
+
+4. On the **Secrets** page, click the **+ Generate/Import** button.
+
+5. In the **Create a secret** blade, enter the following information:
+
+   - **Upload Options** - Select the *Certificate* option.
+
+   - **Upload Certificate** - Select the .pfx certificate to upload.
+
+   - **Name** - The name of the certificate to identify it within the Key Vault.
+
+   - **Activation Date** - Sets when the certificate will become active.
+
+   - **Expiration Date** - Sets when the certificate will become inactive.
+
+   - **Enabled** -Set whether or not the secret data can be retrieved.
+
+   ![Create a new secret](images/azs-browser-add-certificate.png)
+
+6. Click **Create**.
+
+7. Repeat as necessary, depending on how many certificates you require.
+
+### Gathering Key Vault and certificate information
+
+During configuration of the Service Fabric cluster, several details relating to the Key Vault and certificates are required. It is also necessary to configure access to the key vault for Virtual Machines and the Azure Resource Manager.

```

Comment:
During configuration of the Service Fabric cluster, you must provide several details relating to the Key Vault and certificates. You must also configure access to the Key Vault for virtual machines and the Azure Resource Manager.